README.md

```diff
@@ -114,61 +121,144 @@ The milestone needs testing on all platforms. Please report issues when found.
+
+2. **Clone the Descent3 source code.**
 
+    Open a “x64 Native Tools Command Prompt” and run:
     ```batch
     git clone https://github.com/DescentDevelopers/Descent3
+    ```
+
+4. **Build Descent3.**
+
+    ```batch
     cd Descent3
-    cmake --preset win -D ENABLE_LOGGER=[ON|OFF] -D BUILD_EDITOR=[ON|OFF]
+    cmake --preset win
     cmake --build --preset win --config [Debug|Release]
     ```
+    See [Build Options](#build-options) below for more information on `Debug` vs `Release`.
 
 Once CMake finishes, the built files will be put in `builds\win\Descent3\Debug` or `builds\win\Descent3\Release`.
 
-#### Building - macOS
-1. Make sure that [Xcode](https://developer.apple.com/xcode) is installed.
+### Building - macOS
+1. **Install the prerequisite build tools.**
 
-2. Make sure that [Homebrew](https://brew.sh) is installed.
+    * Make sure that [Xcode](https://developer.apple.com/xcode) is installed.
+    * Make sure that [Homebrew](https://brew.sh) is installed.
 
-3. Run these commands:
+2. **Acquire the library dependencies.**
+
+    * If you would like to use vcpkg:
+        ```sh
+        git clone https://github.com/microsoft/vcpkg
+        export VCPKG_ROOT="$PWD/vcpkg"
+        ```
+        **NOTE:**
+        You will need `$VCPKG_ROOT` defined in the environment for all build runs. It is a good idea to set this in your `.bashrc` or equivalent.
+    * If you would like to manage the code dependencies yourself:
+        ```sh
+        brew install sdl2 zlib googletest
+        ```
+
+3. **Clone the Descent3 source code.**
 
     ```sh
     git clone https://github.com/DescentDevelopers/Descent3
+    ```
+
+4. **Build Descent3.**
+
+    ```sh
     cd Descent3
     brew bundle install
-    cmake --preset mac -D ENABLE_LOGGER=[ON|OFF]
+    cmake --preset mac
     cmake --build --preset mac --config [Debug|Release]
     ```
+    See [Build Options](#build-options) below for more information on `Debug` vs `Release`.
 
 Once CMake finishes, the built files will be put in `builds/mac/Descent3/Debug` or `builds/mac/Descent3/Release`.
 
-#### Building - Linux (Ubuntu)
-Run these commands:
+### Building - Linux
+1. **Install the prerequisite build tools.**
 
-```sh
-sudo apt update
-sudo apt install -y --no-install-recommends git ninja-build cmake g++ libsdl2-dev zlib1g-dev
-git clone https://github.com/DescentDevelopers/Descent3
-cd Descent3
-cmake --preset linux -D ENABLE_LOGGER=[ON|OFF]
-cmake --build --preset linux --config [Debug|Release]
-```
+    * APT users (Debian, Ubuntu)
+        ```sh
+        sudo apt update
+        sudo apt install -y --no-install-recommends git ninja-build cmake g++ libncurses-dev
+        ```
+    * DNF users (Red Hat, Fedora)
+        ```sh
+        sudo dnf update --refresh
+        sudo dnf install -y git ninja-build cmake gcc-c++ ncurses-devel
```

Comment:
Why do we need ncurses?

Reply:
#407 removes every use of ncurses

Reply:
ncurses provides `term.h`, which [we do still seem to need](https://github.com/DescentDevelopers/Descent3/blob/main/linux/lnxapp.cpp#L75)

Reply:
Seems this is oversight by me. `term.h` is unused and can be safely removed from code.